Library/Formula/gcc.rb

```diff
@@ -67,6 +70,9 @@ def version_suffix
     version.to_s.slice(/\d\.\d/)
   end
 
+  # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=64089
+  patch :DATA
```

Comment:
Can you please add an explanation of why the patch is needed in a comment in the formula file. Thanks!